src/pages/TeamPage/index.js

```diff
@@ -0,0 +1,349 @@
+import React, { useState, useEffect } from "react";
+import {
+  Button,
+  Typography,
+  Box,
+  Container,
+  Stack,
+  CssBaseline,
+  Card,
+} from "@mui/material";
+import { Grid } from "@mui/material";
+import useMediaQuery from "@mui/material/useMediaQuery";
+import useStyles from "./style";
+import axios from "axios";
+import Tippy from "@tippyjs/react";
+import "./style.css";
+import { breakpoints } from "../../theme/constant";
+import LinkedIn from "../../components/common/SocialMediaIcons/LinkedIn";
+import Twitter from "../../components/common/SocialMediaIcons/Twitter";
+
+/**
+ *
+ * @param {Array} arr
+ * mutates Array inplace
+ */
+function shuffleArray(arr) {
+  for (let i = arr.length - 1; i >= 0; i--) {
+    const newIndex = Math.floor(Math.random() * (i + 1));
+    [arr[i], arr[newIndex]] = [arr[newIndex], arr[i]];
+  }
+  return arr;
+}
+
+function TeamPage() {
+  useEffect(() => {
+    axios({
+      url: `https://anandpatel504.github.io/tarabai-shinde/data/ng_team.json`,
+    }).then((res) => {
+      setTeam(res.data);
+    });
+  }, []);
+  const [team, setTeam] = useState([]);
+  const [members, setMembers] = useState({
+    teamMembers: true,
+    volunteers: false,
+  });
+  const classes = useStyles();
+  const isActive = useMediaQuery("(max-width:" + breakpoints.values.sm + "px)");
+  const isActiveIpad = useMediaQuery("(max-width:1300px)");
+  // console.log(breakpoints.values.sm);
+  console.log(isActive);
+  function Popup(props) {
+    return (
+      <div className={classes.team_descriptionPopup}>
+        {/* <div className="d-flex align-items-center"> */}
+        <div className={classes.team_popupDetails}>
+          <Typography variant="subtitle1" className={classes.team_cardTitle}>
+            {props.Name}
+          </Typography>
+          {props.linkedin ? (
+            <a
+              href={props.linkedin}
+              target="_blank"
+              className={classes.team_socialIcon}
+            >
+              <LinkedIn />
+            </a>
+          ) : (
+            <></>
+          )}
+          {props.twitter ? (
+            <a
+              href={props.twitter}
+              target="_blank"
+              className={classes.team_socialIcon}
+            >
+              <Twitter />
+            </a>
+          ) : (
+            <></>
+          )}
```

Comment:
Use `ExternalLink` here?
https://github.com/navgurukul/bhanwari-devi/blob/47aa4663606dd7ecfd2ab7ab1b93b720f1d029f0/src/components/common/ExternalLink/index.js#L3-L5
Using this and some factoring of commonality:
```suggestion
          {[
                  { Comp: LinkedIn, href: props.linkedin },
                  { Comp: Twitter, href: props.twitter },
                ].map(
                  (socialMedia) =>
                    socialMedia.href && (
                      <ExternalLink
                        href={socialMedia.href}
                        className={classes.team_socialIcon}
                      >
                        <socialMedia.Comp />
                      </ExternalLink>
                    )
                )}
```